tests/testproject/TestProject.AppHost/aspire-manifest.json

```diff
@@ -5,7 +5,9 @@
       "path": "../TestProject.ServiceA/TestProject.ServiceA.csproj",
       "env": {
         "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES": "true",
-        "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES": "true"
+        "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES": "true",
```

Comment:
Are the changes to this and the below file related to this change? They seem unnecessary.

Reply:
This manifest hadn't been updated in a long time. So I regenerated it.